ecs_deploy/cli.py

```diff
@@ -48,6 +48,7 @@ def get_client(access_key_id, secret_access_key, region, profile):
 @click.option('-l', '--log', type=(str, str, str, str), multiple=True, help='Adds or changes a log configuration in the container description (Not available for Fargate): <container> <log driver> <option name> <option value>')
 @click.option('-r', '--role', type=str, help='Sets the task\'s role ARN: <task role ARN>')
 @click.option('-x', '--execution-role', type=str, help='Sets the execution\'s role ARN: <execution role ARN>')
+@click.option('--runtime-platform', type=(str, str), multiple=True, help='Overwrites runtimePlatform: <cpuArchitecture> <operatingSystemFamily>')
```

Comment:
What's the reason this to be defined as `multiple`? Aren't `cpuArchitecture` and `operatingSystemFamily` parameters for the whole task definition and all included containers? The other options are `multiple` because they can be provided per container

Reply:
It was the only way I could get the click library to accept the multi-valued option. After more research into click, I've now committed a fix.